### PR TITLE
Add sources for remaining TSVs

### PR DESCRIPTION
**Pull request by ekpresent**
## Pull Request Description

Add sources for TSVs that don't have them, except those that start with 'HVAC' which will soon be obsolete.

## Checklist

Not all may apply:

- [ ] Unit tests have been added or updated
- [ ] All rake tasks have been run, and pass
- [ ] Documentation has been modified appropriately
- [ ] Any new options are added to `project_testing`
- [ ] `project_testing` runs without any failures
- [ ] No unexpected regression test changes
- [ ] All tests are passing (green) on circleci
- [ ] The [changelog](https://github.com/NREL/OpenStudio-BuildStock/blob/master/CHANGELOG.md) has been updated appropriately
- [ ] This branch is up-to-date with master

For more information on how to perform these checklist items, see the documentation's [Advanced Tutorial](https://resstock.readthedocs.io/en/latest/advanced_tutorial/index.html).

### CODE REVIEW

**Comment by ejhw** [comment]
Thanks for the PR. Looks like this needs the standard '%.6f' formatting that @afontani implemented a while back (https://github.com/NREL/OpenStudio-BuildStock/pull/353). I assume it might be easiest to start with fresh TSVs and manually copy in the new comments, but @afontani @joseph-robertson might have ideas on how to do this more efficiently for ~60 files.

**Comment by joseph-robertson** [comment]
This PR is a workaround until all of these tsv files are created using the tsv maker?

Yes, I would start with fresh tsv files, open them in excel, manually copy in the new comments, and then format numeric fields with decimals equal to six.

**Comment by ejhw** [comment]
Thanks @joseph-robertson. If you start with fresh tsv files, shouldn't that eliminate needing to change numeric formatting (at least if you open in a text editor instead of Excel)?

**Comment by joseph-robertson** [comment]
Yes, if you open in text editor you shouldn't have to reformat the numeric fields. If you open in excel, I've found that when you save it defaults to 2 decimal places.

**Comment by ejhw** [comment]
I guess using something like WinMerge could make it a little faster

**Comment by ekpresent** [comment]
I think I have updated the formatting on all the files I changed to be six digits past the decimal. Anything else on my to-do list for this commit?

File: project_national/housing_characteristics/Insulation Unfinished Attic.tsv [hunk 1]
```diff
@@ -1,451 +1,454 @@
...
+Multi-Family with 5+ Units	CR02	2010s	1	0	0	0	0	0	0	0
+Multi-Family with 5+ Units	CR05	2010s	1	0	0	0	0	0	0	0
+Multi-Family with 5+ Units	CR03	2010s	1	0	0	0	0	0	0	0
+Multi-Family with 5+ Units	CR04	2010s	1	0	0	0	0	0	0	0
+Multi-Family with 5+ Units	CR06	2010s	1	0	0	0	0	0	0	0
+Multi-Family with 5+ Units	CR07	2010s	1	0	0	0	0	0	0	0
+Multi-Family with 5+ Units	CR08	2010s	1	0	0	0	0	0	0	0
+Multi-Family with 5+ Units	CR09	2010s	1	0	0	0	0	0	0	0
+Multi-Family with 5+ Units	CR10	2010s	1	0	0	0	0	0	0	0
+Multi-Family with 5+ Units	CR11	2010s	1	0	0	0	0	0	0	0
```
Comment (ejhw) [inline project_national/housing_characteristics/Insulation Unfinished Attic.tsv:451]: Still needs the decimal fix

File: project_national/housing_characteristics/Refrigerator.tsv [hunk 2]
```diff
@@ -19,6 +19,6 @@ Multi-Family with 2 - 4 Units	Average	0.013683	0.073747	0.322583	0.576304	0.0136
 Multi-Family with 5+ Units	Medium	0.014750	0.061490	0.300636	0.607870	0.014727	475.000000	4721506.000000
```
Comment (ejhw) [inline project_national/housing_characteristics/Refrigerator.tsv:19]: Remove accidental whitespace additions

File: project_national/housing_characteristics/Insulation Unfinished Attic.tsv (hunk 1 above)
Comment (ekpresent) [reply project_national/housing_characteristics/Insulation Unfinished Attic.tsv]: Fixed this one, committed and pushed. Were there others that showed up with the wrong decimals still?

File: project_national/housing_characteristics/Insulation Wall.tsv [hunk 3]
```diff
@@ -1,181 +1,184 @@
...
+CR02	2010s	Masonry	0	0	0	0	0	0	0	0	0	1	0
+CR05	2010s	Masonry	0	0	0	0	0	0	0	0.14	0.14	0.71	0
+CR03	2010s	Masonry	0	0	0	0	0	0	0	0.2	0.05	0.75	0
+CR04	2010s	Masonry	0	0	0	0	0	0	0	0.37	0.32	0.31	0
+CR06	2010s	Masonry	0	0	0	0	0	0	0	0.09	0	0.91	0
+CR07	2010s	Masonry	0	0	0	0	0	0	0	0.39	0.03	0.58	0
+CR08	2010s	Masonry	0	0	0	0	0	0	0	0.6	0.22	0.18	0
+CR09	2010s	Masonry	0	0	0	0	0	0	0	0.67	0.15	0.17	0
+CR10	2010s	Masonry	0	0	0	0	0	0	0	0.41	0.2	0.39	0
+CR11	2010s	Masonry	0	0	0	0	0	0	0	0.47	0.02	0.51	0
```
Comment (ejhw) [inline project_national/housing_characteristics/Insulation Wall.tsv:181]: Still see decimal issue here

File: project_national/housing_characteristics/Neighbors.tsv [hunk 4]
```diff
@@ -4,3 +4,5 @@ Single-Family Detached	1.000000	0.000000	0.000000	0.000000	0.000000	0.000000	0.0
 Single-Family Attached	0.000000	0.005808	0.104524	0.154273	0.286172	0.410100	0.039122
 Multi-Family with 2 - 4 Units	0.000000	0.005808	0.104524	0.154273	0.286172	0.410100	0.039122
 Multi-Family with 5+ Units	0.000000	0.005808	0.104524	0.154273	0.286172	0.410100	0.039122
+# Source: Microsoft Footprint Database (Accessed 2018) for Multi-Family and Single-Family Attached
```
Comment (ejhw) [inline project_national/housing_characteristics/Neighbors.tsv:7]: I mistakenly told you that we used "Microsoft Footprints Database". We have not used that yet. It should read "OpenStreetMap data queried by Radiant Labs"

File: project_national/housing_characteristics/Orientation.tsv [hunk 5]
```diff
@@ -1074,3 +1074,4 @@ WY_Cheyenne.Muni.AP.725640	Multi-Family with 5+ Units	0.165906	0.165906	0.072205
 WY_Green.River-Greater.Green.River.Intergalactic.Spaceport.725744	Multi-Family with 5+ Units	0.165906	0.165906	0.072205	0.072205	0.184840	0.184840	0.077049	0.077049
 WY_Lander-Hunt.Field.725760	Multi-Family with 5+ Units	0.165906	0.165906	0.072205	0.072205	0.184840	0.184840	0.077049	0.077049
 WY_Sheridan.County.AP.726660	Multi-Family with 5+ Units	0.165906	0.165906	0.072205	0.072205	0.184840	0.184840	0.077049	0.077049
+"# Source: Microsoft Footprint Database, accessed 2018"
```
Comment (ejhw) [inline project_national/housing_characteristics/Orientation.tsv:1077]: I mistakenly told you that we used "Microsoft Footprints Database". We have not used that yet. It should read "OpenStreetMap data queried by Radiant Labs"

**Review by ejhw — APPROVED** [review]
Good to resolve conflicts and merge

File: resources/measures/ResidentialAirflow/measure.xml [hunk 6]
```diff
@@ -2,8 +2,8 @@
   <schema_version>3.0</schema_version>
   <name>residential_airflow</name>
   <uid>7d2adf5c-9995-49be-b662-a23d2452bb36</uid>
-  <version_id>a6d7e12f-5ff2-4415-954a-8bc75d4a9e4b</version_id>
-  <version_modified>20201105T190401Z</version_modified>
+  <version_id>a2adcec5-f390-48a7-ad98-9dd0b50fc58b</version_id>
+  <version_modified>20201113T224354Z</version_modified>
```
Comment (ejhw) [inline resources/measures/ResidentialAirflow/measure.xml:6]: The changes to this file look accidental. Probably not a big deal, but make sure to only commit changes related to the PR.

**Review by ejhw — APPROVED** [review]

File: resources/measures/ResidentialAirflow/measure.xml (hunk 6 above)
Comment (joseph-robertson) [reply resources/measures/ResidentialAirflow/measure.xml:6]: This might be ok. Could be result of running `update_measures`.

**Review by joseph-robertson — APPROVED** [review]
👍

**Pull request merged**